packages/pn-personafisica-webapp/public/locales/it/notifiche.json

```diff
@@ -100,7 +103,9 @@
       "not_downloadable_acts": "Poiché sono trascorsi 120 giorni dalla data di perfezionamento, se vuoi ottenere gli atti devi rivolgerti al mittente.",
       "downloadable_aar": "L’avviso di avvenuta ricezione è disponibile online per 10 anni dal perfezionamento della notifica.",
       "not_downloadable_aar": "Poiché sono trascorsi 10 anni dalla data di perfezionamento, l’avviso di avvenuta ricezione non è più disponibile.",
```

Comment:
check if there are old localization that can be removed

Reply:
do it